src/apm.js

```diff
@@ -19,21 +19,35 @@
 
 const { existsSync } = require('fs');
 const { join } = require('path');
-const { name, version } = require('../package.json');
+const { name, version, build } = require('../package.json');
 
-module.exports = function (serviceName = name) {
-  if (process.env.kbnWorkerType === 'optmzr') return;
+/**
+ * flag to disable APM support on all kibana builds by default
+ */
+const isKibanaDistributable = Boolean(build && build.distributable === true);
 
-  const conf = {
+function getConfig(serviceName) {
+  const config = {
     serviceName: `${serviceName}-${version.replace(/\./g, '_')}`
   };
 
-  if (configFileExists()) conf.configFile = 'config/apm.js';
-  else conf.active = false;
+  const configPath = join(__dirname, '..', 'config', 'apm.js');
+  if (existsSync(configPath)) {
+    const configFile = require(configPath); // eslint-disable-line import/no-dynamic-require
+    Object.assign(config, configFile);
```

Comment:
By switching from using the `configFile` config option to manually merging in the config file options, it does change the behavior slightly as the order in which config options are applied is no longer the same. This, however, is probably ok in this case, as we probably want people to overwrite the config options hardcoded in this file, so 👍